glui_clipboard.cpp

```diff
@@ -0,0 +1,303 @@
+/****************************************************************************
+
+  GLUI User Interface Toolkit
+  ---------------------------
+
+     glui_clipboard_x11.cpp - GLUI_Clipboard_Object class
+
+	 This file -is- in the public domain. Circa summer 2017.
+
+      --------------------------------------------------
+
+  Copyright (c) 1998 Paul Rademacher
+
+  This software is provided 'as-is', without any express or implied
+  warranty. In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+  claim that you wrote the original software. If you use this software
+  in a product, an acknowledgment in the product documentation would be
+  appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+  misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+
+  This program is -not- in the public domain.
+
+*****************************************************************************/
+
+#include <limits.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#ifdef GLUI_GLX
+#include <GL/glx.h>
+//Xlib/X crazily pollute the global namespace.
+#include <X11\Xlib.h>
```

Comment:
I don't doubt it. The newer versions of Visual Studio do auto-complete in the `#include` lines. I began by including the X11 headers so I could rough out the code in the IDE (Netbeans is practically unusable for typing even with all of the many many "assistance" features disabled.)

I wouldn't even have typed \ there although in general I do find typing \ easier than / (it can depend on the keyboard.) Today I've been wishing Microsoft would put / in its paths as an option, because if you have to copy-paste them it's real pain to change the slashes.

Bottom line is fix it, and just apply a patch/commit and if it causes issues because a \ is entered, then patch/commit over the /. It's not worth thinking about these things. 

It's not worth turning a change from `(void)` to `()` into a more than 30 seconds ordeal. It took me 15 seconds to make the change in code, and it shouldn't take 15 seconds more to make the change in the code repository. If we take great pains over such minor things then we are both being ridiculous and acting like this is the most important things in our lives--I have to cover much more ground than this kind of quibbling permits. 

When it comes time to do a release, a branch can be locked and fixes can be back-ported to it until it is stable. Stability cannot be a day to day concern.